syntax = "proto3";
package iotamodel;

enum APIResponseType {
    API_STATUS_OK       = 0;
    API_BAD_REQUEST     = 1;
    API_SERVER_ERROR    = 2;
}
message IotaAPIResponse{
    // APIStatus captures call status
    APIResponseType api_status       = 1 ;
    string          error_msg        = 2;
}
